packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.md

```diff
@@ -101,6 +101,74 @@ switch (day) {
 }
 ```
 
+<!--tabs-->
+
+### ❌ Incorrect
+
```

Comment:
it's probably not formatted correctly, would be happy to get some help here

https://deploy-preview-7806--typescript-eslint.netlify.app/rules/switch-exhaustiveness-check#examples